frontends/property/src/styles/main.scss

```
@@ -1,3 +1,99 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lora:wght@600&family=Roboto:wght@100&display=swap');
+$M:768px;
+
+@mixin MQ($canvas){
+  @if $canvas==S{
+      @media only screen and (max-width:$M){
+          @content;
+      }
+  }
+  @else if $canvas==M{
+      @media only screen and (min-width:$M+1){
+          @content;
+      }
+  }
+}
+
 main {
     padding: 1rem;
-}
\ No newline at end of file
+}
+
+.testimonials{
+  display:flex;
+  flex-direction:column;
+  align-items:center;
+  p:first-child {
+    font-family:Roboto;
+    text-transform:uppercase;
+    font-size:1.2rem;
+    padding:1rem;
+    @include MQ(S){
+      font-size:1rem;
+    }
+  }
+  span{
+    font-family:Lora;
+    text-transform:capitalize;
+    font-size:3rem;
+    @include MQ(S){
+      font-size:1.7rem;
+    }
+  }
+  ul {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    max-width:1300px;
+    margin-bottom:3rem;
+    padding:1rem;
+    @include MQ(S){
+      flex-direction:column;
+      padding:0.6rem;
+    }
+    li {
+      @include MQ(S){
+        max-width:100%;
+      }
+      @include MQ(M){
+        max-width:33.33%;
+      }
+    }
+  }
+  button {
```

there is a button.scss, see if you need another one here.

if you do need another button style, put it like this
```suggestion
  button {
     
     &--testimonial {
        // put your styles here
     }
  }
```